src/sass/app_2/_common/components/table.scss

```diff
@@ -0,0 +1,66 @@
+$side-padding: 2rem;
```

Comment:
Seems that the same file already exists with the name [data_table.scss](https://github.com/binary-com/binary-static/blob/new-app/src/sass/app_2/_common/components/data_table.scss)?